pages/roadmap.md

```diff
@@ -286,12 +220,82 @@ This table describes the biggest projects that we’re working on.
         </td>
     </tr>
     <tr>
-        <td><strong>Ktor</strong>
+        <td><strong>Website</strong></td>
+        <td>
+            <ul>
+                <li><a href="https://youtrack.jetbrains.com/issue/KT-44338" target="_blank">Revamp Kotlin documentation</a></li>
+                <li><a href="https://youtrack.jetbrains.com/issue/KT-44339" target="_blank">Make the Kotlin website mobile friendly</a></li>
+                <li><a href="https://youtrack.jetbrains.com/issue/KT-44340" target="_blank">Design a new Kotlin visual style</a></li>
+                <li><a href="https://youtrack.jetbrains.com/issue/KT-44342" target="_blank">Close <em>try.kotlinlang.org</em></a></li>
+            </ul>
         </td>
+        <td></td>
+    </tr>
+    <tr>
+        <td><strong>Ktor</strong></td>
         <td>
            <p><a href="https://blog.jetbrains.com/ktor/2020/08/10/ktor-roadmap-for-2020-2021/" target="_blank">Ktor roadmap</a></p>
          </td>
         <td>
         </td>
     </tr>
-</table>
\ No newline at end of file
+</table>
+
+>* This roadmap is not an exhaustive list of all things the team is working on, only the biggest projects.
+>* There’s no commitment to delivering specific features or fixes in specific versions.
+>* It lists some things that are postponed and will NOT get the team’s attention in the nearest future.
+>* We will adjust our priorities as we go and update the roadmap approximately every three months.
+{:.note}
+
+## What's changed since October 2020
+
+### Completed items
+
+We've **completed** the following items from the roadmap that we published in October 2020:
+
+* ✅ Compiler core: [Bootstrap the new compiler (make the new Kotlin compiler compile itself)](https://youtrack.jetbrains.com/issue/KT-42285)
+* ✅ Kotlin/Native: [Runtime performance: improve object allocation times](https://youtrack.jetbrains.com/issue/KT-42295)
+* ✅ Multiplatform: [KMM plugin: Fix major bugs](https://youtrack.jetbrains.com/issue/KT-42299)
+* ✅ Multiplatform: [KMM plugin: Run common tests on Android devices](https://youtrack.jetbrains.com/issue/KT-42298) (partially)
+* ✅ Libraries: [Improve `kotlinx-coroutines`(release v1.4)](https://youtrack.jetbrains.com/issue/KT-42317)
+
+### Postponed items
+
+We've decided to **postpone** the following items from the previous roadmap: 
```

Comment:
Minor: previous version of the roadmap ?

Reply:
I don't want to use `version` for the roadmap. If it's unclear for users, I'll update phrasing next time. Copyeditiors are OK with the current phrasing.